Composer/packages/client/src/pages/publish/Notifications.tsx

```diff
@@ -56,29 +58,121 @@ export const getPublishedNotificationCardProps = (item: BotStatus): CardProps =>
             name: item.name,
             publishTarget: item.publishTarget,
           }),
-    type: 'pending',
+    type: item.status === 200 ? 'success' : 'error',
     onRenderCardContent: (props) => (
       <div css={cardContent}>
         <Icon css={infoType} iconName="CloudUpload" />
-        <Icon css={statusIconStyle} iconName={item.status === 200 ? 'SkypeCircleCheck' : 'CircleAdditionSolid'} />
+        <Icon css={statusIconStyle} iconName={item.status === 200 ? 'Completed' : 'CircleAdditionSolid'} />
         <div css={cardDetail}>
           <div css={cardDescription}>{props.description}</div>
         </div>
       </div>
     ),
   };
 };
-export const getPendingNotificationCardProps = (items: BotStatus[]): CardProps => {
+
+export const getSkillPublishedNotificationCardProps = (item: BotStatus, url?: string): CardProps => {
+  const skillCardContent = css`
+    display: flex;
+    padding: 0 16px 16px 12px;
+    min-height: 64px;
+  `;
+  const statusIconStyle = css({
+    margin: '12px 0 0 -1px',
```

Comment:
Why negative margin? Seems like you can find a way to avoid it.

Reply:
this is because two icon is too close and we can not change icon size to achieve it, so i use negative margin to do it